ios/FluentUI/Tab Bar/SideTabBar.swift

```diff
@@ -132,16 +132,18 @@ open class SideTabBar: UIView {
 
     private struct Constants {
         static let maxTabCount: Int = 5
-        static let viewWidth: CGFloat = 62.0
-        static let avatarViewTopPadding: CGFloat = 18.0
-        static let avatarViewTopStackViewPadding: CGFloat = 34.0
-        static let bottomStackViewBottomPadding: CGFloat = 14.0
-        static let topItemSpacing: CGFloat = 32.0
-        static let bottomItemSpacing: CGFloat = 24.0
-        static let topItemSize: CGFloat = 28.0
-        static let bottomItemSize: CGFloat = 24.0
-        static let badgeTopSectionPadding: CGFloat = 2.0
-        static let badgeBottomSectionPadding: CGFloat = 4.0
+        static let viewWidth: CGFloat = 62
+        static let avatarViewSafeTopSpacing: CGFloat = 18
+		static let avatarViewMinTopSpacing: CGFloat = 36
+        static let avatarViewTopStackViewSpacing: CGFloat = 34
+        static let bottomStackViewSafeSpacing: CGFloat = 14
+		static let bottomStackViewMinSpacing: CGFloat = 24
```

Comment:
nit: is there inconsistency in indentation?

Reply:
fixed